### PR TITLE
[disaster] add severity filters

### PR DESCRIPTION
**Pull request by chejennifer**
- add ability to filter the events that show up by specified props

screen-recording-_55_ (1)

### CODE REVIEW

File: server/config/disaster_dashboard/Earth.textproto [hunk 1]
```diff
@@ -29,6 +29,7 @@ metadata {
       name: "Earthquake"
       event_type_dcids: "EarthquakeEvent"
       color: "#930000"
+      severity_props: "magnitude"
```
Comment (juliawu) [inline server/config/disaster_dashboard/Earth.textproto:32]: For my own understanding: Are earthquakes the only event type with magnitudes? The severity filter will eventually be applied to things like droughts and hurricanes, right?

Comment (chejennifer) [reply server/config/disaster_dashboard/Earth.textproto:32]: I believe Earthquakes are the only event type with magnitudes right now. Yes, the severity filter will be applied for any event types that we set severity_props for in the config

**Review by juliawu — APPROVED** [review]
Changes look good to me!

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**